core/cmake/BareosSetVariableDefaults.cmake

```diff
@@ -574,15 +574,15 @@ ENDIF()
 
 
 IF(NOT DEFINED job_email)
-   SET(job_email "root@localhost")
+   SET(job_email "root")
 ENDIF()
 
 IF(NOT DEFINED dump_email)
-   SET(dump_email "root@localhost")
+   SET(dump_email "root")
 ENDIF()
```

Comment:
We will need to discuss whether we want this to change.

Reply:
Ok, discuss it please. IMHO it should be either full qualified email address (which is impossible for generally generated package, only if someone compile bareos on his own host, he can enter full email address) or just "root", which should be then correctly completed by MTA. Thank you.